include/mbedtls/ssl.h

```diff
@@ -324,6 +329,9 @@
 #define MBEDTLS_SSL_SESSION_TICKETS_DISABLED     0
 #define MBEDTLS_SSL_SESSION_TICKETS_ENABLED      1
 
+#define MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_DISABLED  0
+#define MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_ENABLED   1
```

Comment:
Naming: we aren't really enabling “new session tickets”, we're enabling the error code that signals that a NewSessionTicket message was received.

Reply:
It is not only enabling the error code, if disabled we do not parse the NewSessionTicket message at all.

Reply:
What does “we do not parse the NewSessionTicket message at all” mean? That we silently discard any message whose header says it's a NewSessionTicket, regardless of whether it's valid or not?

Reply:
For example, if an application calls `mbedtls_ssl_session_save` after each successful call to `mbedtls_ssl_handshake` or `mbedtls_ssl_read`, would it get updated session tickets? I thought it would, since the [code that changes depending on the configuration](https://github.com/Mbed-TLS/mbedtls/pull/9507/files#diff-67127675c5612072a0cb4c12b6dd40b0cfdcc6cdb23839f48ce9e3a2257e77e7) is only code that raises the error, not parsing code.

Reply:
discussed offline, I think we have covered that.